src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -678,6 +680,48 @@ class spell_pri_item_t6_trinket : public SpellScriptLoader
         }
 };
 
+// 60123 - Lightwell
+class spell_pri_lightwell : public SpellScript
+{
+    PrepareSpellScript(spell_pri_lightwell);
+
+    bool Load() override
+    {
+        return GetCaster()->GetTypeId() == TYPEID_UNIT;
+    }
+
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
+    {
+        Creature* caster = GetCaster()->ToCreature();
+        if (!caster || !caster->IsSummon())
+            return;
+
+        uint32 spellId = 0;
+        switch (caster->GetEntry())
+        {
+            case 31897: spellId = 7001; break;
```

Comment:
why not transform those magic numbers in enum?

Reply:
They all indicate the same (Lightwell/Lightwell Renew), and the only difference is level. I am unsure if it is needed to enum each Lightwell1, Lightwell2, and so on.

Reply:
Lightwell_LightwellRenew = 7001 ?

Reply:
Updated PR. Thanks!